src/univariate/continuous/normal.jl

```diff
@@ -131,14 +131,14 @@ function suffstats(::Type{<:Normal}, x::AbstractArray{T}) where T<:Real
     n = length(x)
 
     # compute s
-    s = 0
+    s = zero(T)
     for i in eachindex(x)
         @inbounds s += x[i]
     end
     m = s / n
 
     # compute s2
-    s2 = 0
+    s2 = zero(T)
```

Comment:
This should rather be
```suggestion
    s2 = zero(m)
```
or
```suggestion
    s2 = abs2(zero(m))
```
(I guess the first one should be sufficient): `m` already takes into account and is potentially wider than `T = typeof(x[i])`, and in fact `zero(T)` is "wrong" if eg `T = Int`.